src/components/dashboard/EntityStatOverview/useActiveEntityCount.ts

```diff
@@ -0,0 +1,28 @@
+import { useQuery } from '@supabase-cache-helpers/postgrest-swr';
+import { getLiveSpecShards } from 'api/liveSpecsExt';
+import { isArray } from 'lodash';
+import { useMemo } from 'react';
+import { useTenantStore } from 'stores/Tenant/Store';
+import { Entity } from 'types';
+import { hasLength } from 'utils/misc-utils';
+
+export default function useActiveEntityCount(entityType: Entity) {
+    const selectedTenant = useTenantStore((state) => state.selectedTenant);
+
+    const { data, error, isValidating } = useQuery(
+        hasLength(selectedTenant)
+            ? getLiveSpecShards(selectedTenant, entityType)
+            : null,
+        {
+            refreshInterval: 15000,
+        }
+    );
+
+    const count = useMemo(
+        () =>
+            data ? data.filter(({ disable }) => disable !== true).length : 0,
+        [data]
+    );
+
+    return { count, error, isValidating, loading: !isArray(data) };
```

Comment:
adding a manual `loading` is okay... but does give me a bit of pause since the new SWR also handles validating vs loading and might get confusing.